benefits/settings.py

```diff
@@ -29,13 +29,20 @@ def _filter_empty(ls):
     "django.contrib.messages",
     "django.contrib.sessions",
     "django.contrib.staticfiles",
+    "django_google_sso",  # Add django_google_sso
     "benefits.core",
     "benefits.enrollment",
     "benefits.eligibility",
     "benefits.oauth",
 ]
 
 if ADMIN:
+    GOOGLE_SSO_CLIENT_ID = os.environ.get("GOOGLE_SSO_CLIENT_ID", "secret")
+    GOOGLE_SSO_PROJECT_ID = os.environ.get("GOOGLE_SSO_PROJECT_ID", "benefits-admin")
+    GOOGLE_SSO_CLIENT_SECRET = os.environ.get("GOOGLE_SSO_CLIENT_SECRET", "secret")
+
+    GOOGLE_SSO_ALLOWABLE_DOMAINS = ["compiler.la"]
```

Comment:
@thekaveman Is it okay if the app allows _any_ user with the domain `compiler.la`? Or should it be a specific list of email addresses?

Reply:
Other possible options, depending on what we want:
- Set a certain list of email addresses to be created as superadmin, versus others are just staff permission levels: https://megalus.github.io/django-google-sso/users/#giving-permissions-to-auto-created-users

Reply:
@thekaveman Should this also be an env variable? (List of email addresses that would be superadmins, and everyone else with a compiler email address would be staff).

Reply:
Yeah I think we should start with a specific list of emails (from Compiler) that are allowed in, and ALSO a specific list that gets added as superusers (devs).

Reply:
In this case, I'd recommend we seed the database (save a copy of the DB with the user data) with this pre-configured list of users.

Reply:
We would need some kind of configuration model to be able to put this in the database then, right? Unless the SSO package provides a place for that to be stored?

A site configuration model is something I've been thinking about off and on. But we haven't exactly scoped that or talked about it as a team. So for now, I think an env var that reads from Key Vault via Terraform is the way to go.